aziotctl/src/internal/check/checks/read_certs.rs

```diff
@@ -43,6 +43,7 @@ impl ReadCerts {
         let cert_client = aziot_cert_client_async::Client::new(
             aziot_cert_common_http::ApiVersion::V2020_09_01,
             aziot_certd.clone(),
+            0,
```

Comment:
why 0 retries here?

Reply:
I thought it made sense to not retry during check. If the system is overwhelmed enough to force a retry, I think it's better to fail the check then to pass. It should be extremally rare.